config/initializers/productization.rb

```diff
@@ -1 +1,3 @@
-Vmdb::Productization.new.prepare
+Rails.application.config.to_prepare do
```

Comment:
Yes.  That's what I meant.  Keep in mind, I have no idea what I'm doing when it comes to the asset pipeline.